openmc/material.py

```diff
@@ -311,6 +327,65 @@ def remove_nuclide(self, nuclide):
         if nuclide._name in self._nuclides:
             del self._nuclides[nuclide._name]
 
+    def add_macroscopic(self, macroscopic):
```

Comment:
I agree that a `macroscopic` property would make more sense, especially if we decide to adopt my suggestion of making macroscopic behave as `<material id="1" macroscopic="UO2" />`.